spec/cypress/integration/schools/core_induction_programme/ChooseMaterials.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Induction tutors choosing programmes
+  Background:
+    Given cohort was created with start_year "2021"
+    And school was created with name "Really awesome school"
+    And school_cohort was created with created cohort, created school and induction_programme_choice "core_induction_programme"
+    And core_induction_programme was created with name "Awesome induction course"
+    And I am logged in as an induction coordinator for created school
```

Comment:
I think I would put this in a scenario. It would be easier, right?

Reply:
Much easier but also much longer. We need some better way of creating seed data for scenarios..